### ISSUE TITLE
Normalize UI

### ISSUE THREAD

**Issue by ivan-kleshnin** [Paqmind/html-to-hyperscript#7]

**Comment by scabbiaza** [comment]
Updated styles: https://github.com/Paqmind/html-to-hyperscript/pull/8/files

![screen shot 2016-01-01 at 11 59 01](https://cloud.githubusercontent.com/assets/2734464/12070473/1be6371e-b07f-11e5-9788-2b264fa05ea3.png)

Removed scroll bars by default: it's impossible to control neither width nor colors. 
Because design has a lot of gray colors, uncontrolled gray color from scroll adds messy.
I have added gray left padding for panels, so now code is not so close to the borders.
I've also updated styles for footer.

Examples with scroll:
![screen shot 2016-01-01 at 12 02 10](https://cloud.githubusercontent.com/assets/2734464/12070479/911880be-b07f-11e5-9d48-00069e0490dd.png)
![screen shot 2016-01-01 at 12 02 20](https://cloud.githubusercontent.com/assets/2734464/12070480/9333dbaa-b07f-11e5-982e-56dd395ff92b.png)
